.github/actions/smoke-test/check-image-size.sh

```
@@ -0,0 +1,60 @@
+    IMAGE_SIZE=$(docker image inspect --format='{{.Size}}' "$IMAGE_ID")
+    # Output the size
+    echo "Size of the image $IMAGE_ID: $IMAGE_SIZE bytes"
+    threshold=$(convert_gb_to_bytes "$THRESHOLD_IN_GB")
+    # Retrieve the Docker image size
+    echo -e "\nThreshold is $threshold bytes ie $THRESHOLD_IN_GB GB"
+    # Remove the 'MB' from the size string and convert to an integer
+    image_size=${IMAGE_SIZE%bytes}
+    image_size=${image_size//.}
+    # Check if the image size is above the threshold
+    echo -e "\n🧪 Checking image size of $IMAGE :"
+    if [ -n $image_size  ] && [ $image_size -gt $threshold ]; then
```

Shellcheck (0.10.0)

[error] -n doesn't work with unquoted arguments. Quote or use [[ ]].

(SC2070)